source/cl/source/command_queue.cpp

```diff
@@ -52,6 +52,7 @@ _cl_command_queue::_cl_command_queue(cl_context context, cl_device_id device,
       in_flush(false) {
   cl::retainInternal(context);
   cl::retainInternal(device);
+  { device->RegisterCommandQueue(this); }
```

Comment:
The extra braces here are surprising to me, do they have a non-obvious function or are they accidental?